cmd/lotus-storage-miner/auth.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+
+	"gopkg.in/urfave/cli.v2"
+
+	"github.com/filecoin-project/lotus/api/apistruct"
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var authCmd = &cli.Command{
+	Name:  "auth",
+	Usage: "Manage RPC permissions",
+	Subcommands: []*cli.Command{
+		authCreateAdminToken,
+	},
+}
+
+var authCreateAdminToken = &cli.Command{
+	Name:  "create-token",
+	Usage: "Create token",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:  "perm",
+			Usage: "permission to assign to the token, one of: read, write, sign, admin",
+		},
+	},
+
+	Action: func(cctx *cli.Context) error {
+		napi, closer, err := lcli.GetStorageMinerAPI(cctx)
```

Comment:
We could avoid duplicating this by putting this command on the `api.Common` interface, and using `GetAPI` which picks the correct thing